operator/controller/src/main/java/io/apicurio/registry/operator/util/Util.java

```diff
@@ -0,0 +1,11 @@
+package io.apicurio.registry.operator.util;
+
+public class Util {
+
+    private Util() {
+    }
+
+    public static boolean isBlank(String value) {
```

Comment:
nitpick: this is used exclusively in `IngressUtil`, move this method to be private in the using class.